content/en/blog/2024/otel-docs-survey/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Insights from the OpenTelemetry Docs Usability Survey
+linkTitle: OTel-docs Survey
+date: 2024-12-13
+author: '[Tiffany Hrabusa](https://github.com/tiffany76) (Grafana Labs)'
+issue: https://github.com/open-telemetry/opentelemetry.io/issues/5793
+sig: Communications, End-User
+cSpell:ignore: Hrabusa
+---
+
+[The OpenTelemetry End-User SIG](/community/end-user/) recently surveyed the
+community to find out how user-friendly [OpenTelemetry's documentation](/docs/)
+is. In an earlier survey, two-thirds of respondents named comprehensive
+documentation as a top resource they wished they'd had when getting started with
+OpenTelemetry. So we decided to dig a little deeper. The Docs Usability Survey
+asked users where they go for OTel documentation, what they'd like to see more
+of in the docs, and how they rate the current state of the docs. We received 48
+responses, which we'll use to focus our documentation efforts and help us
+improve in key areas.
+
+Check out the results!
+
+## Key takeaways
+
+- Folks want more visual aids like diagrams and screenshots.
+- Of the three types of documentation we asked about (component concepts,
+  installation instructions, and troubleshooting), the troubleshooting docs need
+  the most work.
+- When asked what information they'd most like to see added to OTel's docs, the
+  top answers were **more examples** and **more coverage**, both in depth and
+  breadth.
+- The Collector docs were the most consulted docs set based on number of
+  ratings. This result tallies with the page view analysis in the SIG
+  Communications'
+  [year-end review](../year-in-review/#which-pages-were-the-most-popular).
+- After normalization and weighting, the
+  [Java documentation](/docs/languages/java/) was the overall highest rated docs
+  set, suggesting that the efforts made to
+  [improve its information architecture](../year-in-review/#ia-improvements)
+  have paid off. Meanwhile, the [Swift](/docs/languages/swift/) docs received
+  the overall lowest rating.
+- Among the six most popular docs sets, the [JavaScript](/docs/languages/js/)
+  docs received the lowest rating.
```

Comment:
How about the following:

```suggestion
- Respondents expressed a desire for **more visual aids**, such as diagrams and
  screenshots.
- Of the three types of documentation we asked about (component concepts,
  installation instructions, and troubleshooting), the **troubleshooting docs**
  were identified as needing the most improvement.
- When asked about the information they’d most like to see added to OTel's docs,
  the top responses were **more examples** and **expanded coverage**, both in
  depth and breadth.
- The [Collector](/docs/collector/) docs emerged as the most frequently
  consulted resource, a finding that aligns with the page view analysis in the
  SIG Communications'
  [year-end review](../year-in-review/#which-pages-were-the-most-popular).
- After normalization and weighting, the [Java](/docs/languages/java/)
  documentation received the **highest overall rating**, reflecting the positive
  impact of recent improvements to its
  [organization](../year-in-review/#ia-improvements). Conversely, the
  [Swift](/docs/languages/swift/) docs received the lowest overall rating.
- Among the six most popular documentation sets, the
  [JavaScript](/docs/languages/js/) docs received the **lowest rating**.
```

Reply:
If the highest ratings are takeaways, I think the lowest ones are too. I think we should keep them, but I'm also fine removing them if you think we should.